src/typeahead/highlight.ts

```diff
@@ -30,28 +30,30 @@ export class NgbHighlight implements OnChanges {
    * The text highlighting is added to.
    *
    * If the `term` is found inside this text, it will be highlighted.
+   * If the `term` contains array then all the items from it will be highlighted inside the text.
    */
-  @Input() result: string;
+  @Input() result: any;
 
   /**
-   * The term to be highlighted.
+   * The term or array of terms to be highlighted.
    */
-  @Input() term: string;
+  @Input() term: any;
```

Comment:
Please use the appropriate type → `string | string[]`

Reply:
how about `string | string[] | number | number[]`? will ensure test coverage for all options